src/scan_shadow_detector.cpp

```diff
@@ -0,0 +1,75 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ * 
+ *  Copyright (c) 2017-2021, laser_filters authors
```

Comment:
Same comment here - let's leave radu explicitly since he did the initial implementation, and add "other authors" in addition.

Reply:
Radu's name was not in original header file of scan_shadow_detector.h. In this file only laser_filter authors were mentioned in header comments. Underneath, Atsushi Watanabe was mentioned as author in separate comment with \author annotation.

So for new cpp file I just copied the original header from scan_shadow_detector.h